lib/tf_lock/tf_lock_acquire.py

```diff
@@ -70,11 +70,11 @@ def tf_lock_acquire(root_module: Path) -> ExitCode:
 
 
 # TODO: deduplicate wrt spec.gh.pr.PR.is_closed
-def is_pr_closed(gh_url: str) -> bool:
+def is_pr_unlockable(gh_url: str) -> bool:
     status = sh.stdout(
         ("gh", "pr", "view", gh_url, "--json", "state", "--jq", ".state")
     )
-    return status in ("CLOSED", "MERGED")
+    return status in ("CLOSED", "MERGED", "DRAFT")
```

Comment:
It's not that simple, unfortunately:

```suggestion
def is_pr_unlockable(gh_url: str) -> bool:
    status = sh.json(
        ("gh", "pr", "view", gh_url, "--json", "state,isDraft")
    )
    ...
```